tour_operator/src/main/kotlin/pl/edu/pg/rsww/tour_operator/TourOperatorServer.kt

```diff
@@ -0,0 +1,27 @@
+package pl.edu.pg.rsww.touroperator
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.springframework.amqp.rabbit.annotation.Queue
+import org.springframework.amqp.rabbit.annotation.RabbitListener
+
+public class TourOperatorServer {
+    @RabbitListener(queuesToDeclare = [Queue("#{queueConfig.requests}")])
+    fun requestHandler(request: String): String {
```

Comment:
Should this be using API gateway's message types rather than their own? Billing is something that is handled by the server and I don't think there will be a need to expose anything from TO as an HTTP API to the client. There are some additional fields (status code, headers map) that wouldn't be needed + we could just have the payment result directly in the message type rather than in the response body.

Reply:
There are no benefits of designating separate message type to tour operator service. Current approach would allow for simpler and more uniform communication between services.